infra-parent/infra-v2/src/main/java/com/redhat/service/smartevents/infra/v2/api/models/dto/BridgeDTO.java

```diff
@@ -123,6 +127,14 @@ public void setOperationType(OperationType operationType) {
         this.operationType = operationType;
     }
 
+    public ZonedDateTime getDeletionRequestedAt() {
```

Comment:
@vaibhavjainwiz The `operation` property will be set to `DELETE` . I think that gives us enough?

Reply:
done